src/applications/vaos/tests/new-appointment/components/DateTimeSelectPage/index.unit.spec.jsx

```diff
@@ -1,201 +1,226 @@
 import React from 'react';
 import { expect } from 'chai';
 import sinon from 'sinon';
-import { mount, shallow } from 'enzyme';
 import moment from 'moment';
+import { renderWithStoreAndRouter } from '../../../mocks/setup';
+import userEvent from '@testing-library/user-event';
 
 import {
   DateTimeSelectPage,
```

Comment:
Same comment as on the other page, we should use the version of this component that's connected to Redux.